cli/crates/server/src/udf_builder.rs

```diff
@@ -18,20 +18,29 @@ async fn run_command<P: AsRef<Path>>(
     environment: &[(&'static str, &str)],
 ) -> Result<(), JavascriptPackageManagerComamndError> {
     let command_string = format!("{command_type} {}", arguments.iter().format(" "));
-
-    trace!("running '{command_string}'");
+    let current_directory = current_directory.as_ref();
+    if !current_directory.exists() {
```

Comment:
In most cases we do `try_exists()` to handle cases of missing permissions, I'd do that here as well

Reply:
good call